src/main/resources/db/migration/changes/nnn-Instructors-create.json

```
@@ -0,0 +1,29 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "Instructors-create",
+        "author": "your-name",
```

```suggestion
        "id": "008-Instructors-create",
        "author": "andrewkim102,
```

Also please rename the `nnn` in the filename to `008` if indeed that is the next number in sequence.